### DISCUSSION TITLE
Survey on Pinecil Tips - please do the survey

### DISCUSSION THREAD

**Discussion by River-Mochi** [Ralim/IronOS#1465] [General]
There is a Pinecil Tips Survey here: https://forms.gle/ahKKMapCpHvLzZYX8

Due to the huge response to the Pine64 subreddit post about changing the Pinecil included tip (currently conical B2), some community members made a Survey to gather community feedback. The idea is to give the Pine64 Store some data so they get a general idea of what most people want.

- If you care about changing the default Tip from conical, please fill out the survey.
- If you love or hate the Conical tip, fill out this survey.

This [survey](https://docs.google.com/forms/d/e/1FAIpQLSdpVm3QVYEFk-dWVnWL_PnLYPgab-Y4Vus1K7U_O7Xcm0PCNg/viewform) is important to see if changes could be made to the Included tip and if the 4-packs could be improved since they are currenly missing some popular/useful tips like the BC3 tip. 

Another issue mentioned by many in pine64 reddit and elsewhere is the large e-waste.  I myself received a B2 conical tip with all 4 of my pinecils, and all 4 B2 tips sit unused in a box since I got better, more general purpose tips (BC2/BC3/D24/Ku) that are easier to use.  Although I was able to get all 11 types of tips and experiment with them, other people are not as fortunate and need their 1 included/free tip to do more.  As Pine Store does not allow selection of the included tip, here is your chance to give feedback on which tip this should be.

This survey may not change anything or it may be the data needed to get some long requested changes.

**Comment by voluptuary** [comment]
Ku for LIFE!

**Reply by River-Mochi** [reply]
your vote has been noted :)  thanks for doing the survey
